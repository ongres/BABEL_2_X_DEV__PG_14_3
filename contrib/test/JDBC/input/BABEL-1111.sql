-- test connectionproperty() function
-- invalid property name, should return NULL
select connectionproperty('invalid property');
GO

select connectionproperty(NULL);
GO

-- valid supported properties
select connectionproperty('net_transport'),connectionproperty('protocol_type'), connectionproperty('auth_scheme'), connectionproperty('local_tcp_port');
GO

select @@MICROSOFTVERSION;
GO