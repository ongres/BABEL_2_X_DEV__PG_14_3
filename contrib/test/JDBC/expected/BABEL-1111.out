-- test connectionproperty() function
-- invalid property name, should return NULL
select connectionproperty('invalid property');
GO
~~START~~
sql_variant
<NULL>
~~END~~


select connectionproperty(NULL);
GO
~~START~~
sql_variant
<NULL>
~~END~~


-- valid supported properties
select connectionproperty('net_transport'),connectionproperty('protocol_type'), connectionproperty('auth_scheme'), connectionproperty('local_tcp_port');
GO
~~START~~
sql_variant#!#sql_variant#!#sql_variant#!#sql_variant
TCP#!#TSQL#!#SQL#!#1433
~~END~~


select @@MICROSOFTVERSION;
GO
~~START~~
int
201332885
~~END~~

